.github/workflows/documentation_accessibility_checks.yml

```diff
@@ -66,19 +71,11 @@ jobs:
 
   start_slack_thread:
     name: Start Slack thread
-    runs-on: ubuntu-latest
     if: ${{ failure() }}
     needs: axe-scan
-    steps:
```

Comment:
Is this OK to remove?

Reply:
Yes. Now it is using start_slack_thread.yml instead. It is ok. Other Workflow are using the same format as well.

Reply:
Can you break this work off into another PR if it isn't related to the chromium issue?

Reply:
yes, I rolled it back